OPSEXP-139 Document custom images overrides (#855)
Alfresco/acs-deployment: docs/helm/README.md

![Helm Deployment Community](./diagrams/helm-community.png)

> ## What to expect from the ACS Helm charts

Reviewer: ```suggestion
## Overview
```

> Alfresco provides tested Helm charts as a "deployment template" for customers
> who want to take advantage of the container orchestration benefits of
> Kubernetes. These Helm charts are undergoing continual development and
> improvement, and should not be used "as is" for your production environments,
> but should help you save time and effort deploying Alfresco Content Services for
> your organisation.

Reviewer: ```suggestion
Alfresco provides tested Helm charts as a "template" to accelerate deployment and configuration for customers
who wants to take advantage of Kubernetes orchestration capabilities. Please remember that as stated in our [support policies](https://docs.alfresco.com/support/latest/policies/deployment/), similarly to other deployment artifacts, they are not meant to be used 'as-is' in production.

Our goal is to save you time and effort deploying Alfresco Content Services for
your organization. 
```

> Helm charts values contains secrets to be set. For deployment in sensitive
> environments please see the [Security](security.md) section.

Reviewer: ```suggestion
## Using the Helm charts

To set your secrets please see the [Security](security.md) section, especially for deployment in sensitive
environments
```

> The Helm charts in this repository provide a PostgreSQL database via [Bitnami
> charts](https://github.com/bitnami/charts) and do not configure any logging or

Reviewer: ```suggestion
Out of the box we provide a PostgreSQL database via [Bitnami
charts](https://github.com/bitnami/charts) and do not configure any logging or
```

monitoring. This design has been chosen so that they can be installed in a
Kubernetes cluster without changes and are still flexible to be adapted to your
actual environment.

> For your environment, you should use these charts as a starting point and
> extends them so that ACS integrates into your infrastructure. You typically want
> to remove the PostgreSQL container and connect the cs-repository directly to
> your database.

Reviewer: ```suggestion
You typically want
to remove the PostgreSQL container and connect the cs-repository directly to
your database.
```

![Helm Deployment Community](./diagrams/helm-community.png)

## Overview

Alfresco provides tested Helm charts as a "template" to accelerate deployment

> and configuration for customers who wants to take advantage of Kubernetes

Reviewer 2: ```suggestion
and configuration for customers who want to take advantage of Kubernetes
```
